packages/raystack/v1/components/avatar/avatar.tsx

```diff
@@ -1,7 +1,7 @@
 import * as AvatarPrimitive from "@radix-ui/react-avatar";
 import { cva, VariantProps } from "class-variance-authority";
 import { clsx } from 'clsx';
-import { Box } from "~/box";
+import { Box } from "../box";
```

Comment:
Cross version imports were throwing cyclic dep error thus moved some components locally and imported.